docs/_source/tutorials/notebooks/training-llm-mistral-sft.ipynb

```diff
@@ -0,0 +1,946 @@
+      "\n",
+      "When was Bertrand Russell born?\n",
+      "<bot>: 1872\n",
+      "</s><s><human>: When did Bertrand Russell die?\n",
+      "<bot>: 1970\n",
+      "</s><s><human>: What did Frege say about math?\n",
+      "<bot>: all mathematical truths are logical\n",
+      "</s><s><human>: What did Wittgenstein say about logic?\n",
+      "<bot>: all logical truths are mere linguistic tautologies\n",
+      "</s><s><human>: What are pseudoproblems?\n",
+      "<bot>: most metaphysical, ethical, aesthetic and other traditional philosophical problems\n",
+      "</s><s><human>: When was Frege's thesis published?\n",
+      "<bot>: Sorry, I cannot find a relevant answer in the given context.\n",
+      "</s><s><human>: When was Wittgenstein born?\n",
+      "<bot>: Sorry, I cannot find a relevant answer in the given context.\n",
+      "</s><s><human>: When did Wittgenstein die?\n",
+      "<bot>: Sorry, I cannot find a relevant answer in the given context.\n",
+      "</s><s><human>: Who was a neopositivist?\n",
+      "<bot>: Sorry, I cannot find a relevant answer in the given context.\n",
+      "</s><s><human>: What gave a sentence meaning? <bot>: Sorry, I cannot find a relevant answer in the given context.</s>\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(formatted_dataset[80][\"text\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This seems to match our desired format exactly!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data Collation\n",
+    "Next, we want to verify that this text is tokenized and collated correctly. In particular the data collation can be quite important, as some data collators will replace all special tokens (e.g. padding, BOS and EOS tokens) with a label of -100, which is the label that is ignored during training. If this is the case, then the model will not learn to produce the EOS token. Consequently, the model will always respond with endless tokens - far from ideal.\n",
```

Comment:
This is indeed part of the context I mentioned above but maybe a general intro would be nice.

Reply:
Handled in 2838cccc43d0e713b041c8c876df8211cc1f5999, I rephrased this slightly to make the intro a bit more smooth